### PR TITLE
#130 scope suffix

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
v0.2.2

### CODE REVIEW

File: src/providers/CompletionItemProvider.ts [hunk 1]
```diff
@@ -180,10 +180,13 @@ const getPythonSymbols = async (isFunction: boolean, lineText: string, position:
   const modules = Object.values(modulesByUri);
   modules.push(l10n.t("New module name"));
   cis.push(
-    getCompletionItemInString(isFunction ? l10n.t("create a new function") : l10n.t("create a new class"), lineText, position, [
-      modules.length === 1 ? modules[0] : modules,
-      isFunction ? l10n.t("function name") : l10n.t("class name"),
-    ], getPythonSuffix(isFunction))
+    getCompletionItemInString(
+      isFunction ? l10n.t("create a new function") : l10n.t("create a new class"),
```
Comment (FabienLelaquais) [inline src/providers/CompletionItemProvider.ts]: Is all lower case ok in the context?

**Review by FabienLelaquais — APPROVED** [review]
Confidence is everything

File: src/providers/CompletionItemProvider.ts (hunk 1 above)
Comment (FredLL-Avaiga) [reply src/providers/CompletionItemProvider.ts]: it's the last option of the completion list ...

Comment (FredLL-Avaiga) [reply src/providers/CompletionItemProvider.ts]: image

Comment (FredLL-Avaiga) [reply src/providers/CompletionItemProvider.ts]: What do you think ?

Comment (FredLL-Avaiga) [reply src/providers/CompletionItemProvider.ts]: Create it will be

**Pull request merged**